akka-docs/rst/scala/http/routing-dsl/directives/debugging-directives/logRequestResult.rst

```diff
@@ -30,3 +30,13 @@ Example
 
 .. includecode2:: ../../../../code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala
    :snippet: logRequestResult
+
+
+Example for logging response time.
+----------------------------------
+
+This example will showcase the advance logging using the DebuggingDirective. 
```

Comment:
The following example showcases building advanced logging directives on top of the provided `DebuggingDirectives`.